src/level/level.cpp

```diff
@@ -238,22 +247,112 @@ namespace level {
         artifacts_.remove(i);
     }
 
+    void Level::enemyShoot(entities::Shooter *s, manager::Level *levelManager){
+        Collidable *collision;
+        bool firable = true;
```

Comment:
firable = sparabile, mi dice poco riguardo a cosa deve fare questo booleano. 

A quanto mi sembra di aver capito serve per capire se hai la linea di tiro libera, quindi puoi chiamarlo come 
`canShoot` oppure `isLineOfFireClear` o qualcosa che mi dica in che modo lo utilizzi riguardo al fire.